paymentsheet/src/main/java/com/stripe/android/paymentsheet/addresselement/AddressUtils.kt

```diff
@@ -47,3 +54,18 @@ internal fun AddressDetails.editDistance(otherAddress: AddressDetails?): Int {
     editDistance += (address?.state ?: "").levenshtein(comparedAddress?.state ?: "")
     return editDistance
 }
+
+@Composable
+internal fun ScrollableColumn(
+    modifier: Modifier = Modifier,
+    content: @Composable ColumnScope.() -> Unit
+) {
+    Box(
+        modifier = Modifier.verticalScroll(rememberScrollState())
+    ) {
+        Column(
+            modifier = modifier,
+            content = content
+        )
+    }
+}
```

Comment:
Similar to what we’re using in Link. Let me know if you prefer a different approach.